upgrade_release_version.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+
+function usage() {
+	echo "  -h help"
+	echo "  -o <oldversion>"
+	echo "  -n <newversion>"
+}
+
+function replace() {
+
+declare -a fileList=("README.md" "pom.xml" "src/main/java/com/adyen/Client.java")
+for FILENAME in "${fileList[@]}"
+
+do
+  XMLTMPFILE=$FILENAME.bak
+  echo "Updating file - $FILENAME"
+  cat $FILENAME > $XMLTMPFILE
+  sed -i.bak s/${OLDVERSION}/${NEWVERSION}/g $FILENAME
+  rm -f $XMLTMPFILE
+done
+}
+
+while getopts "ho:n:" o; do
+    case $o in
+        h)
+           usage
+            ;;
+        o)
+          OLDVERSION=${OPTARG};;
+
+         n)
+          NEWVERSION=${OPTARG};;
+    esac
+done
+
+echo "Old release version is" $OLDVERSION
+echo "New release version is"$NEWVERSION
+
+if [ -z "${OLDVERSION}" ]; then
+    echo "Please specify the 'oldversion' argument";
```

Comment:
I would not call it old version but current version (with a space in between)

Reply:
this is argument name so there is no space here.